docs/docs/featurizer/semantic_map.md

```diff
@@ -0,0 +1,27 @@
+The `SemanticMapFeaturizer` is an experimental *sparse* featurizer developed by Rasa.
+It can only be used in combination with pre-trained embedding files, which you can
+find [here](https://github.com/RasaHQ/rasa-embeddings/tree/main/embeddings/semantic_map).
+Please refer to our [blog posts](blog.rasa.com/exploring-semantic-map-embeddings-1/) for more details.
```

Comment:
1. Is this blog live yet? 
2. Let's add https. blog.rasa.com/exploring-semantic-map-embeddings-1/ -> https://blog.rasa.com/exploring-semantic-map-embeddings-1/